poetry/core/masonry/builders/builder.py

```diff
@@ -332,6 +341,9 @@ def __hash__(self):
     def __repr__(self):  # type: () -> str
         return str(self.path)
 
+    def relative_to_project_root(self):  # type(): -> Path
+        return self.path.relative_to(self.project_root)
```

Comment:
Should we care about absolute path includes? These might throw an exception here since we do not explicitly disallow them yet?

Reply:
Absolute path includes are not supported yet, correct? So if someone has an absolute path in the `include` they get problems anyway :)